File: js/Findthedivisors!.js
// Copilot: ignore file
// Kata: Find the divisors!
// Level: 7kyu
// Solved: 2025-08-14

function divisors(integer) {
    let res = [];
    for (let i = 2; i < integer; i++) {
        if (integer % i === 0) {
            res.push(i);
        }
    }
    if (res.length === 0) {
        res = `${integer} is prime`;
    }
    return res;
}

File: python/Findthedivisors!.py
# Copilot: ignore file
# Kata: Find the divisors!
# Level: 7kyu
# Solved: 2025-08-14

def divisors(integer):
    res = []

    for i in range(integer):
        if i > 1 and integer % i == 0:
            res.append(i)

    if len(res) == 0:
        res = f"{integer} is prime"

    return res 